keyboards/levinson/keymaps/treadwell/keymap.c

```diff
@@ -160,12 +160,12 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
       }
       return false;
       break;
-    case DVORAK:
+    case GAME:
       if (record->event.pressed) {
         #ifdef AUDIO_ENABLE
           PLAY_SONG(tone_dvorak);
         #endif
-        persistent_default_layer_set(1UL<<_DVORAK);
+        persistent_default_layer_set(1UL<<_GAME);
```

Comment:
All of these could be replaced with: 
https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

And you could remove the audio defines above, and put them in your keymap's config.h